share/completions/minikube.fish

```diff
@@ -0,0 +1,79 @@
+complete -c minikube -f -n "__minikube_no_command" -a service -d "Gets the kubernetes URL(s) for the specified service in your local cluster"
+complete -c minikube -f -n "__minikube_no_command" -a ssh -d "Log into or run a command on a machine with SSH; similar to 'docker-machine ssh'"
+complete -c minikube -f -n "__minikube_no_command" -a start -d "Starts a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a status -d "Gets the status of a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a stop -d "Stops a running local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a version -d "Print the version of minikube."
+complete -c minikube -l "alsologtostderr" -d "log to standard error as well as files"
+complete -c minikube -l "log_backtrace_at" -d "when logging hits line file:N, emit a stack trace (default :0)"
+complete -c minikube -l "log_dir" -d "If non-empty, write log files in this directory (default \"\")"
+complete -c minikube -l "logtostderr" -d "log to standard error instead of files"
+complete -c minikube -l "show-libmachine-logs" -d "Deprecated: To enable libmachine logs, set --v=3 or higher"
+complete -c minikube -l "stderrthreshold" -d "logs at or above this threshold go to stderr (default 2)"
+complete -c minikube -s "v" -d "log level for V logs"
+complete -c minikube -l "v" -d "log level for V logs"
+complete -c minikube -l "vmodule" -d "comma-separated list of pattern=N settings for file-filtered logging"
+complete -c minikube -n "__minikube_using_command addons" -l "format" -d "Go template format string for the addon list output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/"
+complete -c minikube -n "__minikube_using_command addons" -l "format" -d "Format to output addons URL in.  This format will be applied to each url individually and they will be printed one at a time. (default \"http://{{.IP}}:{{.Port}}\")"
+complete -c minikube -n "__minikube_using_command addons" -l "https" -d "Open the addons URL with https instead of http"
+complete -c minikube -n "__minikube_using_command addons" -l "url" -d "Display the kubernetes addons URL in the CLI instead of opening it in the default browser"
+complete -c minikube -n "__minikube_using_command config" -l "format" -d "Go template format string for the config view output.  The format for Go templates can be found here: https://golang.org/pkg/text/template/"
+complete -c minikube -n "__minikube_using_command dashboard" -l "url" -d "Display the kubernetes dashboard in the CLI instead of opening it in the default browser"
+complete -c minikube -n "__minikube_using_command docker-env" -l "no-proxy" -d "Add machine IP to NO_PROXY environment variable"
+complete -c minikube -n "__minikube_using_command docker-env" -l "shell" -d "Force environment to be configured for a specified shell: [fish, cmd, powershell, tcsh, bash, zsh], default is auto-detect"
```

Comment:
These can be given as arguments, no? So `-a "fish cmd powershell tcsh bash zsh auto-detect"`? Then you could remove them from the description.